plugins/org.python.pydev/src/org/python/pydev/ui/pythonpathconf/conda/PyDevCondaPreferences.java

```diff
@@ -0,0 +1,28 @@
+package org.python.pydev.ui.pythonpathconf.conda;
+
+import java.io.File;
+
+import org.python.pydev.core.IInterpreterInfo.UnableToFindExecutableException;
+import org.python.pydev.core.preferences.PydevPrefs;
+import org.python.pydev.ui.pythonpathconf.package_manager.CondaPackageManager;
+
+public final class PyDevCondaPreferences {
+    private static final String CONDA_EXEC_PATH = "CONDA_EXEC_PATH";
+    private static final String DEFAULT_CONDA_EXEC_PATH = "";
+
+    public static final String getExecutablePath(CondaPackageManager condaPackageManager)
```

Comment:
I think that these setters would be a bit better receiving a File and returning a File and the getter should do a validation if the file exists (if the preference is set and the file doesn't exist it should throw a UnableToFindExecutableException).

Reply:
So the setter should not have a file existance validation?

Reply:
And the setter should receive a file, set file path in cache and then return the same file?

Reply:
Yes, the setter doesn't need a validation (the getter needs to do it anyways -- for instance, it's possible that the user uninstalls conda and we have to handle that).

The getter should always read from the preference, convert the value to a `File`, validate and then return it.

Reply:
Update it in the last commit ;)